src/main/scala/consul/v1/event/EventRequests.scala

```diff
@@ -20,7 +20,9 @@ object EventRequests{
 
   private implicit lazy val eventReads = Json.reads[Event]
 
-  def apply(basePath: String)(implicit executionContext: ExecutionContext):EventRequests = new EventRequests {
+  def apply(basePath: String)(implicit executionContext: ExecutionContext, rb: ConsulRequestBasics):EventRequests = new EventRequests {
+
+    import rb._
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/Codacy/scala-consul/file/1255759086/issues/source?bid=2788492&fileBranchId=2939115#l25)